tests/Github_.py

```diff
@@ -1,16 +1,15 @@
 ############################ Copyrights and license ############################
 #                                                                              #
-# Copyright 2012 Vincent Jacques <vincent@vincent-jacques.net>                 #
```

Comment:
removing these lines does not look right

Reply:
Yup, that was done by the automated script. I can manually fix it, but as I noted in the description anyone that runs the script will just revert it. It looks like this file was renamed a few years ago and the script does not follow the rename backwards, so it only sees anyone who touched the new file.

Reply:
Fixed in #2728, you can ignore this.

Reply:
Ignore this as in "these changes are fine to commit as is" or, "revert and just add myself and they'll be updated once that PR is merged"?

Reply:
Fine to ignore, they will be restored in #2801.